Ethan's Edits (#148)
teachee-capstone/teachee: reports/final/report.tex

together the functionality most required in Electrical Engineering labs and packages
it with lightweight and portable software. 

TeachEE's hardware consists of a four-layer custom designed Printed Circuit
Board (PCB) powered by a USB connection to the computer. The PCB has a
two-channel 1MSPS ADC for reading low-speed signals and current from a

> hall-effect sensor. The PCB also has a two-channel 40MSPS Analog to Digital

Reviewer: 40MSPS should be separated with space

Reviewer: Same with 1MSPS

Author: Fixed

\subsection{Hardware} % ETHAN
The hardware solution for TeachEE is implemented in the form of a four-layer
PCB. The PCB is designed using the Electronic Design Automation (EDA) software
Altium Designer, the industry standard tool for PCB design. Our design takes

> advantage of Altium's hierarchical schematics. This allows the schematic to be

Reviewer: ```suggestion
advantage of Altium's hierarchical schematics, allowing the schematic to be
```

The FPGA module schematic had one mistake that was not discovered until the
physical PCBs had already been manufactured and delivered. To use the
parallel-bus interface of the FT232HQ to send data, the FPGA must lock to a
\SI{60}{\mega\hertz} output clock from the FT232HQ. During layout, the FPGA pin

> wired to this clock was swapped to a normal GPIO pin rather than an Multi Region

Reviewer: ```suggestion
wired to this clock was swapped to a normal GPIO pin rather than a Multi Region
```

In the online electrical engineering labs during the pandemic, an electronics
professor noticed issues with the kits provided to students. There was a general
lack of quality and brittleness which caused students to require replacement

> kits. Additionally, the scopes and signal generators had few protections against

Reviewer: Best to not use "scopes" and just "oscilloscopes"

Author: I agree my bad there

> extraneous inputs resulting in students accidentally breaking their tools when

Reviewer: Comma after "inputs"

Author: Fixed

> incorrectly connected. At a functional level, the scopes were missing critical

Reviewer: scopes

Author: fixed

> features. They were single channel and had no ability to measure current or

Reviewer: ```suggestion
features: they were single channel and had no ability to measure current or
```
